frappe/public/js/frappe/form/templates/form_sidebar.html

```diff
@@ -54,17 +54,45 @@
 	<li class="sidebar-label attachments-label">
 		<svg class="icon icon-sm"><use href="#icon-attachment"></use></svg>
 		{%= __("Attachments") %}
+
+		<li class="explore-full-btn hidden">
+			<button class="data-pill btn">
+				<span class="pill-label ellipsis">
+					{%= __("Explore Files") %}
+				</span>
+				<svg class="icon icon-sm">
+					<use href="#icon-projects"></use>
+				</svg>
+			</button>
+		</li>
+
+		<li class="attachments-actions">
+			<button class="data-pill btn add-attachment-btn">
+				<span class="pill-label ellipsis">
+					{%= __("Attach File") %}
+				</span>
+				<svg class="icon icon-sm">
+					<use href="#icon-add"></use>
+				</svg>
+			</button>
+
+			<button class="text-muted btn btn-default icon-btn explore-btn">
+				<svg class="icon icon-sm">
+					<use href="#icon-projects"></use>
+				</svg>
+			</button>
+		</li>
+
+
+		<li class="show-all-btn hidden">
+			<button class="data-pill btn">
+				<span class="pill-label ellipsis">
+					{%= __("Show All") %}
+				</span>
+			</button>
+		</li>
```

Comment:
```suggestion
		<li class="show-all-btn hidden">
			<a href="" class="pill-label ellipsis">
				{%= __("Show All") %}
			</a>
		</li>
```

This button feels a bit too prominent/distracting to me. What do you think about formatting it as a link?

![Bildschirmfoto 2023-08-04 um 16 40 55](https://github.com/frappe/frappe/assets/14891507/0ce1a453-0547-40cf-8b36-11d946492070)